packages/TelegramClient-UI.package/TCUTelegram.class/instance/openCurrentWindow.st

```diff
@@ -0,0 +1,10 @@
+drawing
+openCurrentWindow
+
+	| currentCenter |
+	
+	currentCenter := self center.
+	self extent: self currentWindow extent.
+	self center: currentCenter.
```

Comment:
why `currentCenter`? does `extent` move the morph?

Reply:
Yeah sadly `extent:` uses the topleft corner I think as a reference point. While writing this comment I had another idea: just doing `self bounds: (self currentWindow extent center: self center).`.
Would be better right?